ci_checks.sh

```diff
@@ -31,7 +31,7 @@ cargo make check --all-features --locked &&
 cargo make check --locked &&
 OVERRIDE_CHAIN_CONFIGS=true cargo test --test integration_tests local_node &&
 cargo nextest run --workspace &&
-FUEL_ALWAYS_USE_WASM=true cargo nextest run --all-features --workspace &&
+cargo nextest run --all-features --workspace &&
```

Comment:
What's the rationale behind this change on this PR ?

Reply:
A mistake. Removed in b3ea6058e1.

Reply:
Huh I think you changed an other line instead of removing this change.

Reply:
Not sure what happeded here. Well, now I forced the file to match latest `master` in a0f34f68f6